src/components/core/Flyout/Flyout.recipe.ts

```diff
@@ -4,7 +4,7 @@ import { defineSlotRecipe } from "@pandacss/dev";
 export const flyoutRecipe = defineSlotRecipe({
   className: "flyout",
   description: "The styles for the Flyout component",
-  slots: [...popoverAnatomy.keys(), "closeTriggerIcon"],
+  slots: [...popoverAnatomy.keys()],
```

Comment:
Can remove array syntax here as well as the spread operator now since we no longer need to append the `closeTriggerIcon` as an additional slot.